File: agent/api/src/main/java/me/jiahuan/openrc/agent/pojo/DeviceInfo.java
package me.jiahuan.openrc.agent.pojo;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class DeviceInfo {
    private String deviceId;
    private String manufacturer;
    private String model;
    private String os;
    private String osVersion;
    private Size size;
}

File: device/app/src/main/java/me/jiahuan/openrc/device/background/device/DeviceInfo.kt
package me.jiahuan.openrc.device.background.device

import me.jiahuan.openrc.device.background.model.Size

/**
 * 设备信息
 */
class DeviceInfo(
    var manufacturer: String,
    var model: String,
    var os: String,
    var osVersion: String,
    var size: Size
)